packages/core/src/2d/assembler/TiledSpriteAssembler.ts

```diff
@@ -139,7 +147,7 @@ export class TiledSpriteAssembler {
     renderer._bounds.transform(worldMatrix);
   }
 
-  static updateUVs(renderer: SpriteRenderer): void {
+  static updateUVs(renderer: SpriteRenderer | UIImage): void {
```

Comment:
Same as commented above.
we have renderer: SpriteRenderer | UIImage, which includes only two classes.
If we anticipate this pattern expanding or being reused in the future, it might still be beneficial to define a type like Renderable for consistency and maintainability, even though there are currently only two classes. However, if this combination is unique to this specific case, leaving it as-is might be more straightforward.